tools/cockpit.spec

```diff
@@ -617,8 +615,8 @@ The Cockpit component for managing storage.  This package uses udisks.
 
 %package -n cockpit-tests
 Summary: Tests for Cockpit
-Requires: cockpit-bridge >= 138
-Requires: cockpit-system >= 138
+Requires: cockpit-bridge >= %{required_base}
+Requires: cockpit-system >= %{required_base}
```

Comment:
Interesting, that raises base from 138 -> 266, but that should be fine for `cockpit-tests`

Reply:
Right, it mostly just needs to be satisfied in the RHEL cockpit/appstream split, but that's been the case for some time. This is mostly to get rid of magic numbers.